include/mbedtls/error.h

```diff
@@ -75,11 +78,11 @@
  * X509      2   20
  * PKCS5     2   4 (Started from top)
  * DHM       3   9
```

Comment:
Didn't adding MBEDTLS_ERR_DHM_HW_ACCEL_FAILED increase the DHM error count by 1?

Reply:
Indeed. Thanks, fixed.